runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1880,6 +1882,12 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 		return JNI_EINVAL;
 	}
 
+	if (extensions->userSpecifiedParameters._checkpointGCThreads._wasSpecified && extensions->gcThreadCount > 0) {
+		if (extensions->checkpointGCthreadCount > extensions->gcThreadCount) {
+			j9nls_printf(PORTLIB, J9NLS_WARNING, J9NLS_GC_THREAD_VALUE_USER_MUST_BE_AT_LEAST_CHECKPOINTGCTHREAD_WARN, extensions->checkpointGCthreadCount);
+		}
+	}
+
```

Comment:
This piece of the code looks the on the wrong place, we should find better location for it.

Reply:
Agreed, it's probably cleaner to be a part of (just after) parsing of the checkpoint option.

But for that to work, we have to make sure that parsing for all different ways (-Xgcthreads, -Xgc:threads=, etc)  to specify ext->gcthreads occur before parsing for the checkpoint option. And then we should add a comment in the code that we rely on it.

Reply:
As we discussed meanwhile, it's also ok to move it to the new method in ConfigurationDelegate.

Reply:
I've updated a new version for this, which handled this situation in ConfigurationDelegate